package-lock.json

```diff
@@ -0,0 +1,2226 @@
+{
+  "name": "identity-reporting",
+  "version": "0.0.0",
+  "lockfileVersion": 1,
```

Comment:
Up for consideration whether it's worth getting on NPM 7 now vs. later, which will reformat this as `"lockfileVersion": 2`. I can't recall if this is default version with current Node LTS, though LTS is also bumping to 16 in October.

Reply:
🤷 if you think it's worthwhile, happy to give that a shot? what do we have to change in the package.json to make that happen?

Reply:
Just need to run `npm install` after upgrading. Which is be partly why it might be good to do now, since it tends to be confusing for someone already running npm 7 when it introduces local changes simply by installing 😬

Reply:
Should we also pin an node version in package.json?

Reply:
> Should we also pin an node version in package.json?

I don't think `engines` will do much for us in terms of encouragement or enforcement. Doesn't hurt to include though. And maybe an `.nvmrc` file would be useful?

Reply:
Ok! Do you mind making a PR with a version you think would make sense?

Reply:
Sure 👍  See #4.